src/GrpcProtos/capture.proto

```diff
@@ -236,51 +236,165 @@ message ApiStringEvent {
 }
 
 message ApiTrackInt {
+  // NextID: 14
   int32 pid = 1;
   int32 tid = 2;
   uint64 timestamp_ns = 3;
 
   int32 data = 4;
+
+  // We encode the first 64 characters of the scope's name/label using fixed64
```

Comment:
I think we should have this comment once in `ApiScopeStart` and all the other cases should refer to that comment.

Reply:
Done.